core/state/statedb_test.go

```diff
@@ -515,15 +410,16 @@ func (test *snapshotTest) checkEqual(state, checkstate *StateDB) error {
 		checkeq("GetCode", state.GetCode(addr), checkstate.GetCode(addr))
 		checkeq("GetCodeHash", state.GetCodeHash(addr), checkstate.GetCodeHash(addr))
 		checkeq("GetCodeSize", state.GetCodeSize(addr), checkstate.GetCodeSize(addr))
+		// XXX: Can this be restored?
```

Comment:
It's commented out since the `forEachStorage` requires iterating on storage and we don't have that ability over the public interface as far as I can tell.